### PR TITLE
Fix pipeline

### PR DESCRIPTION
**Pull request by trojanowski**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -34,9 +34,11 @@ jobs:
         id: cache
         with:
           path: .venv
-          key: poetry-${{ hashFiles('**/poetry.lock') }}
+          # The `2` suffix is used because there is no way to manually clear
```
Comment (Wauplin) [inline .github/workflows/main.yml]: Does this means that each time we want to clear cache, we need to update this number ? If yes, I would add a comment about it. If not, i'm not sure what has been done here.

Comment (trojanowski) [reply .github/workflows/main.yml]: @Wauplin it's also evicted automatically if not used for 7 days: <https://docs.github.com/en/free-pro-team@latest/actions/guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy>. I'll add a comment about it.

Comment (trojanowski) [reply .github/workflows/main.yml]: @Wauplin updated.

Comment (Wauplin) [reply .github/workflows/main.yml]: Thanks for the explanation ! :)

**Review by Wauplin — APPROVED** [review]

**Pull request merged**